upstream-community-operators/synopsys/synopsys-operator.2019.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,479 @@
+    mediatype: image/png
+  keywords: [security monitoring]
+  version: 2019.4.0
+  maturity: stable
+  maintainers:
+  - name: Synopsys, Inc
+    email: software-integrity-support@synopsys.com
+  provider:
+     name: Synopsys, Inc
+  labels:
+     alm-owner-etcd: synopsys-operator
+     alm-status-descriptors: synopsys-operator.2019.4.0
+  selector:
+     matchLabels:
+       alm-owner-etcd: synopsys-operator
+       alm-status-descriptors: synopsys-operator.2019.4.0
+  links:
+   - name: Synopsys Integrity
+     url: https://www.synopsys.com/software-integrity.html
+   - name: Synopsys Operator Documentation
+     url: https://synopsys.atlassian.net/wiki/spaces/BDLM/pages/34373652/Synopsys+Operator
+   - name: Black Duck Datasheet
+     url: https://www.synopsys.com/content/dam/synopsys/sig-assets/datasheets/blackduck-sca-ds-ul.pdf
+   - name: OpsSight Data Sheet
+     url: https://www.synopsys.com/content/dam/synopsys/sig-assets/datasheets/openshift-ds-ul.pdf
+   - name: Black Duck Alert Documentation
+     url: https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/622882/Black+Duck+Alert
+   - name: Synopsys Operator Source Code
+     url: https://github.com/blackducksoftware/synopsys-operator
+  installModes:
+   - type: OwnNamespace
+     supported: true
+   - type: SingleNamespace
+     supported: false
+   - type: MultiNamespace
+     supported: false
+   - type: AllNamespaces
+     supported: true
+  install:
+    strategy: Deployment
```

Comment:
Please use lowercase `deployment` for `spec.install.strategy`

Reply:
Done